### ISSUE TITLE
second thermostat not showing

### ISSUE THREAD

**Issue by TomMalfrere** [FilipDem/Domoticz-NEST-plugin#43]
Hi,
I have a nest thermostat and a nest protect at home and they work perfectly with your plugin in Domoticz.

However, I added a second NEST thermostat on a second location to my google account.
I have a separate Domoticz running on the second location and I want to pull in the NEST thermostat.
I managed to get the issueToken and Cookie filled in OK but I can only see the first thermostat and not the second one.
Is there a way to select the second thermostat?

**Comment by FilipDem** [comment]
Unfortunately I have no 2 NESTs? Are it the same? The Nest Thermostat E is not supported.
You can uncomment the dump in file nest.py at line 259-260. This will give you a dump of all the information that his returned by the Nest server. Then you can analyse if your second thermostat is also in...
If it is in the dump, then there is probably a problem with the parsing to find all the thermostats in line 265-266...
But the starting point is looking at the return of the Nest server.
Filip

**Comment by TomMalfrere** [comment]
I have 1 nest at home and I added a second 'home' where there is also a nest.
Your plugin does detect the NEST thermostat and NEST protect of my first home.
But it seems the second 'home' and/or it's devices are not detected. 
I suspect the parsing of multiple homes is not implemented.

However, I decided to use a second google account for the second home.
Why? 
1) because then, you don't have to implement anything extra
2) even if you parse it, It does not seem correct to me that I would get info in Domoticz from another location...
